website/static/js/contribManager.js

```diff
@@ -49,20 +49,30 @@ var setupEditable = function(elm, data) {
     });
 };
 
-var ContributorModel = function(contributor, pageOwner, isRegistration) {
+var ContributorModel = function(contributor, manager, pageOwner, isRegistration, isAdmin) {
```

Comment:
Or.. maybe instead of passing the the `ContributorsViewModel`, just pass `currentUserCanEdit` (which will come from `ContributorsViewModel#canEdit()`. That way, you won't have to delete `serialized.manager` on L68, and `ContributorModel` doesn't have to know anything about `ContributorsViewModel`